src/components/button/button.css

```
@@ -1,8 +1,24 @@
-.btn {
-  border-radius: 20px;
-  font-size: 1.2rem;
-  color: white;
-  background: #016b6b;
-  border: none;
-  padding: 10px 20px;
+.btn-home-grid {
+  display: grid;
+  grid-auto-flow: column;
+  justify-content: center;
+  align-content: center;
+  width: 138px;
+  height: 55px;
+  border-radius: 27pt;
+  background-color: var(--bg-color);
+}
+
+.btn-home-img {
+  color: var(--secondary-color);
+  padding-right: 15px;
+  height: 20px;
```

height is given from picture

```suggestion
  color: var(--secondary-color);
  padding-right: 15px;
```

---

It's actually a small change, barely noticeable.
The height value is taken from the XD file. It's a really small difference, wouldn't make a real difference, I think